[2022-07-22 12:59:44,639] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-07-22 12:59:45,508] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-07-22 12:59:45,513] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:45,517] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 12:59:45,518] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:45,851] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-03 00:00:00+00:00
[2022-07-22 12:59:46,029] {standard_task_runner.py:52} INFO - Started process 743 to run task
[2022-07-22 12:59:46,110] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpjtjkallj', '--error-file', '/tmp/tmpszp40wko']
[2022-07-22 12:59:46,138] {standard_task_runner.py:80} INFO - Job 21: Subtask t2
[2022-07-22 12:59:46,447] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 12:59:47,293] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-03T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 12:59:48,054] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-07-22 12:59:48,184] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 12:59:48,603] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220603T000000, start_date=20220722T125944, end_date=20220722T125948
[2022-07-22 12:59:48,857] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 12:59:50,479] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:17:53,908] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-07-25 09:17:54,295] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-03T00:00:00+00:00 [queued]>
[2022-07-25 09:17:54,300] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:54,304] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:17:54,317] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:54,576] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-03 00:00:00+00:00
[2022-07-25 09:17:54,611] {standard_task_runner.py:52} INFO - Started process 698 to run task
[2022-07-25 09:17:54,723] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-03T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpme5l3o7v', '--error-file', '/tmp/tmpyuwkzv3j']
[2022-07-25 09:17:54,752] {standard_task_runner.py:80} INFO - Job 17: Subtask t2
[2022-07-25 09:17:55,030] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:17:55,838] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-03T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:17:56,359] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-03T00:00:00+00:00
[2022-07-25 09:17:56,366] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:17:56,566] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220603T000000, start_date=20220725T091753, end_date=20220725T091756
[2022-07-25 09:17:56,708] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:17:56,836] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
